scifight/migrations/0001_initial.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.9.7 on 2016-07-17 03:22
```

Comment:
No strong need to include this file into a commit if only one thing changed is a timestamp in a comment.